### PR TITLE
[DISCOVERY-240] - Attach debugger to running container

### PR DESCRIPTION
**Pull request by bruno-fs**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,9 +14,11 @@ services:
       QPC_SERVER_USERNAME: admin
       NETWORK_CONNECT_JOB_TIMEOUT: 6
       NETWORK_INSPECT_JOB_TIMEOUT: 600
+      QPC_DEBUGPY: 1
```
Comment (gtanzillo) [inline docker-compose.yml:17]: Should this default to `0`, assuming `0` means off?

Comment (bruno-fs) [reply docker-compose.yml:17]: yeah, zero means off!

I set to `1` considering docker-compose is meant for developers, not end-users :thinking: 

But if you prefer I can change anyway. its easy to override docker-compose locally without actually editing the file :thinking:

Comment (gtanzillo) [reply docker-compose.yml:17]: No, that makes sense, let's keep it `1`. Thanks for the clarification

**Review by gtanzillo — APPROVED** [review]
lgtm 👍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,9 +39,46 @@ You can run the lint command on your branch to check compliance.
 make lint
 ```
 
+### Debugging
+
+You can easily attatch a debugger to the running qpc-server on docker compose, which
```
Comment (abellotti) [inline CONTRIBUTING.md:44]: attach

**Pull request merged**